### DISCUSSION TITLE
y blockUi is working in DEV environment but not in Production Environment

### DISCUSSION THREAD

**Discussion by Alirzrezaei** [primefaces/community#1697] [PrimeFaces] [answered]
I am using Primefaces 13 with a combination of JBoss 7.4 and Java 11. 

I used a blockUi to block the entire form until a bulk transaction is done (may take more than 10 Mins).

The code was working fine with Primefaces 6, JBoss 7.2, and Java 8, in all environments.  but after upgrading to the above-mentioned versions, the block is working in the DEV environment until the end of the transaction, but in the Production environment, the loading panel disappears after some minutes. However, the process is not finished and is running in the background. 

`onstart="PF('blockUi2').show()" oncomplete="PF('blockUi2').hide()"`

` <p:blockUI block="data" id="mqp-blockui"  
		    widgetVar="blockUi2" >
		 <img style="height:8em;background-color:transparent !important;"
			src="#{facesContext.externalContext.requestContextPath}/themes/default/pics/loading.gif" />
	</p:blockUI>`

**Comment by melloware** [comment]
Are you using PrimeFaces 13.0.8 ?   If so and its still happening can you please create a small reproducer using the [PrimeFaces Test](https://github.com/primefaces/primefaces-test) project.  It is the only way developers can debug your problem to help.

**Reply by Alirzrezaei** [reply, marked as answer]
I inspected the page and when the below error occurs, the blockUI loading panel disappears. 

Failed to load resource: the server responded with a status of 502 (Bad Gateway).

Then in Preview TAB, I see the below explanation:

The proxy server received an invalid response from an upstream server.
The proxy server could not handle the request
Reason: Error reading from remote server

**Reply by melloware** [reply]
Yep that is a networking issue on your end you need to look into. Not BlockUi related
